tests/test_config.py

```diff
@@ -586,10 +587,11 @@ def test_merge(merge_log_err, hass):
     config_util.merge_packages_config(hass, config, packages)
 
     assert merge_log_err.call_count == 0
-    assert len(config) == 5
+    assert len(config) == 6
```

Comment:
Add a test where you're reloading automation config and it picks up packages, since that's what you're fixing

Reply:
As there's tests already for merging then in which case wouldn't it pick up the automations?

Reply:
You are not interacting with the entity component here, which is where you made your change. You need to write a test that hits your changed code path.